programs/mpl-core/src/state/collection.rs

```diff
@@ -98,11 +98,27 @@ impl CollectionV1 {
         CheckResult::None
     }
 
+    /// Check permissions for the add external plugin lifecycle event.
+    pub fn check_add_external_plugin() -> CheckResult {
+        CheckResult::CanApprove
+    }
+
+    /// Check permissions for the remove external plugin lifecycle event.
+    pub fn check_remove_external_plugin() -> CheckResult {
+        CheckResult::CanApprove
+    }
+
+    /// Check permissions for the update external plugin lifecycle event.
+    pub fn check_update_external_plugin() -> CheckResult {
+        CheckResult::None
+    }
```

Comment:
So just for right now nobody can update external plugins?

Reply:
Oh good catch.

Reply:
Actually I take it back. The approve comes from the plugin validation.